churn_server/churn_server.ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
<p>@yjb-ds </p><ol><li>no need for <code>str(clf_model_file)</code>  since clf_model_file is a local file name</li><li>extra_data is a dict of DataItems , so u shouldnt do str() on it, will not work w remote files, need to use <code>cox_model_file.local()</code>  , same for <code>km_model_file</code>  </li><li>i dont see that you use any of those extra models </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I will be using the models